src/consumer/component/sources/files.js

```diff
@@ -1,16 +1,16 @@
 import fs from 'fs-extra';
 import path from 'path';
+import Vinyl from 'vinyl';
 import Source from './source';
 import FileSourceNotFound from '../exceptions/file-source-not-found';
 import { isString } from '../../../utils';
 import logger from '../../../logger/logger';
 
-export type FileSrc = { name: string, content: string|Buffer };
 
 // TODO: Remove Source?
-export default class Files extends Source {
-  constructor(src: FileSrc[]) { // eslint-disable-line
-    super(src);
+export default class Files extends Vinyl {
```

Comment:
I think of changing this class to SourceFile or something like this.
I don't like the plural form.
What do you think?